spec/services/partner_fetch_requestable_items_service_spec.rb

```diff
@@ -4,8 +4,24 @@
   describe '#call' do
     subject { described_class.new(partner_id: partner_id).call }
     let(:partner_id) { partner.id }
-    let(:partner) { create(:partner) }
-    let(:organization) { partner.organization }
+    let(:partner) { create(:partner, organization:) }
+    let(:organization) { create(:organization, skip_items: true, items:) }
+    let(:items) {
+      [
+        build(:item, active: true, visible_to_partners: true),
+        build(:item, active: false, visible_to_partners: true),
+        build(:item, active: true, visible_to_partners: false),
+        build(:item, active: false, visible_to_partners: false)
+      ]
+    }
+    let(:partner_items) {
+      [
+        build(:item, active: true, visible_to_partners: true),
+        build(:item, active: false, visible_to_partners: true),
+        build(:item, active: true, visible_to_partners: false),
+        build(:item, active: false, visible_to_partners: false)
+      ]
+    }
```

Comment:
Aren't these exactly the same? You can probably just do `let(:partner_items) { items.deep_dup }`